src/components/Chip.stories.tsx

```diff
@@ -58,3 +58,15 @@ export function WithIcon() {
     </div>
   );
 }
+
+export function WithStartAdorment() {
+  return (
+    <div css={Css.wPx(200).$}>
+      <Chip startAdornment={<span css={Css.smBd.$}>K</span>} text="Kitchen" />
```

Comment:
@JonnCharpentier this is fine, but I was thinking maybe we could just change `text: string` to `text: ReactNode`, similar to how `title: ReactNode`.

That way we could do like `text=...snippet of html...`

Can we change to that instead? It seems lighter weight / more flexible than a new prop.

Reply:
For the Chip that could work, but for the `ToggleChipGroup` we need the `options.label` to be a string so we can use it for `aria-label`.